packages/common/src/content/edit.ts

```diff
@@ -168,6 +169,12 @@ export async function updateContent(
     }
   }
 
+  if (content.schedule.mode === "automatic") {
+    await deleteSchedule(item, requestOptions);
+  } else if (content.schedule !== (await getSchedule(item, requestOptions))) {
+    await setSchedule(item, content.schedule, requestOptions);
+  }
```

Comment:
Couple things to consider:
- What does `deleteSchedule()` do if no schedule is actually set? If it throws a 4xx error we'll need to account for that
- `content.schedule` is an object, so `!==` probably won't work the way you're expecting. You'll either need to create a deep compare function _or_ just always save the schedule.